tmail-backend/blob/blobid-list/src/test/java/com/linagora/tmail/blob/blobid/list/SingleSaveBlobStoreContract.scala

```diff
@@ -0,0 +1,63 @@
+package com.linagora.tmail.blob.blobid.list
+
+import com.linagora.tmail.blob.blobid.list.SingleSaveBlobStoreDAOFixture._
+import org.apache.james.blob.api.{BlobId, BucketName, ObjectStoreException}
+import org.assertj.core.api.Assertions.{assertThat, assertThatThrownBy}
+import org.assertj.core.api.ThrowableAssert.ThrowingCallable
+import org.junit.jupiter.api.Test
+import reactor.core.scala.publisher.SMono
+
+trait SingleSaveBlobStoreContract {
+
+  def singleSaveBlobStoreDAO: SingleSaveBlobStoreDAO
+
+  def blobIdList: BlobIdList
+
+  def blobIdFactory: BlobId.Factory
+
+  def defaultBucketName: BucketName
+
+  @Test
+  def saveShouldSuccessIfBlobIDoesNotExist(): Unit = {
+    val blobId: BlobId = blobIdFactory.randomId()
+    SMono.fromPublisher(singleSaveBlobStoreDAO.save(TEST_BUCKET_NAME, blobId, SHORT_BYTEARRAY)).block()
+    val isStore: Boolean = SMono.fromPublisher(blobIdList.isStored(blobId)).block()
+    assert(isStore)
```

Comment:
```suggestion
    assertThat(SMono.fromPublisher(blobIdList.isStored(blobId)).block()).isTrue
```

Use AssertJ assertThat for your assertions.